Classes/UIScrollView+InfiniteScroll.m

```diff
@@ -499,6 +569,10 @@ - (void)pb_scrollViewDidScroll:(CGPoint)contentOffset {
  *  Scrolls down to activity indicator position if activity indicator is partially visible
  */
 - (void)pb_scrollToInfiniteIndicatorIfNeeded {
+    
+    // top
+    if (self.infiniteScrollDirection == UIScrollViewInfiniteScrollDirectionTop) return;
```

Comment:
Indicator can be half visible, this is when we need to scroll up or down to reveal or hide it based on scroll position. It's similar to how search bar works in table views.